lib/presentation/profile/profile_bloc.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:bloc/bloc.dart';
+import 'package:meta/meta.dart';
+import 'package:stronk/api/user_repo.dart';
+import 'package:stronk/domain/model/user.dart';
+
+import '../../auth_manager.dart';
+
+class ProfileState {
+
+  final User user;
+
+
+  ProfileState({@required this.user});
+
+  @override
+  String toString() {
+    return ("\n name: ${user.name}");
+  }
+}
+
+@sealed
+abstract class _Event {}
+
+class InitEvent implements _Event {}
+
+
+class ProfileBloc extends Bloc<_Event, ProfileState> {
```

Comment:
Do we have tests for this class? I know this is renamed but i don't see any changes in tests :P